tsl/src/continuous_aggs/planner.h

```diff
@@ -10,5 +10,6 @@
 #include "planner/planner.h"
 
 void constify_cagg_watermark(Query *parse);
+void cagg_sort_pushdown(Query *parse, int *cursor_opts);
```

Comment:
Shouldn't this also be enclosed by an `#if PG16_GE`?

Reply:
idk no harm by it not being enclosed, i was actually thinking about removing the ifdef in the source file too cause they are just noise and its sufficient to have this check in one place